scene/main/window.h

```diff
@@ -31,10 +31,10 @@
 #ifndef WINDOW_H
 #define WINDOW_H
 
+#include "scene/gui/control.h"
```

Comment:
Why was this added?

Reply:
Residue from the previous implementation, removed.